com.chartboost.mediation/Editor/ChartboostMediationSettingsEditor.cs

```diff
@@ -1,10 +1,9 @@
-using System;
 using UnityEditor;
 using UnityEngine;
 
-namespace Helium.Editor
+namespace Chartboost.Editor
```

Comment:
```suggestion
namespace ChartboostMediation.Editor
```

This will conflict with a legacy Chartboost Unity Editor. Let us use ChartboostMediation